README.md

```diff
@@ -49,13 +49,13 @@ jobs:
 
 ## Versioning
 
-Using the versioned semantic [tags](#Tags) is recommended for security and reliability.
+Using the versioned semantic [tags](#tags) is recommended for security and reliability.
```

Comment:
This change is not correct. The anchor name has a capital.

Reply:
@julien-carsique-sonarsource did you try?

I did, and it actually works as expected; that issue was reported by the markdown linter.